es/customization/customization.md

```diff
@@ -8,4 +8,4 @@
 * [Modelos](models.md)
 * [Componentes](components.md)
 * [Gemas](gems.md)
-* [Adaptar la aplicación](overwritting.md)
+* [Adaptar la aplicación](overwriting.md)
```

Comment:
With this change, the name of the Spanish file would need to be updated from `overwritting.md` to `overwriting.md`. The same as has been done for the English file.

Reply:
I'll do it.